README.rst

```diff
@@ -75,34 +52,84 @@ Install collective.indexvocabularies by adding it to your buildout::
 
 and then running ``bin/buildout``
 
+Activation
+----------
+
+Activate the add-on by installing it in the Addons Control panel.
+
+Configure
+---------
+
+To create a vocabulary from an index, go to the 'Index Vocabularies' control
+panel and select the indexes you wish to create vocabularies for.
+
+To use the vocabulary in a tag field, you will need to edit the XML Schema of
+your content type. Currently this is only possible via the Classic UI
+(Control Panel -> Content Types -> Schema -> Edit XML Schema)
```

Comment:
```suggestion

To create a custom tag field on a content type you will need to do the following

1. Create a new field of type "Multiselect"
2. Save the content type. This will create an index for this field.
3. Go to Dynamic Vocabularies in the control panel
4. Select the Index matching the field you just created and click "Make Vocabulary".
5. Copy the name of the vocabulary generated.
8. Edit your content type again but this time via the editing it's XML
   - Currently this is only possible via the Classic UI (Control Panel -> Content Types -> Schema -> Edit XML Schema)
9. Change the widget to a TagSelectWidget with the vocabulrary name that was generated.  

 e.g. 
    <field name="test" type="zope.schema.Set">
      <title>test</title>
      <value_type type="zope.schema.TextLine" />
      <form:widget type="collective.indexvocabularies.widgets.TagSelectWidget">
          <vocabulary>collective.dynamicvocabularies.uniquevalues.test</vocabulary>
      </form:widget>
    </field>
  
```

Reply:
why does this include our own widget? The standard one should work right?